sleap_nn/config/data_config.py

```diff
@@ -0,0 +1,166 @@
+import attrs
+from omegaconf import MISSING
+from typing import Optional, Tuple, List, Dict, Any
+
+
+"""Serializable configuration classes for specifying all data configuration parameters.
+
+These configuration classes are intended to specify all 
+the parameters required to initialize the data config.
+"""
+
+
+@attrs.define
```

Comment:
Here we can import define directly from attrs and use
```suggestion
@define
```